.gitignore

```diff
@@ -123,6 +123,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.vscode/
```

Comment:
Hey! I run into this all the time as a VS Code user - some projects don't like to have IDE-specific stuff in their `.gitignore` (I don't know how @ceteri feels about this, but still), but you can use the `.git/info/exclude` file. With other words, you can add `.vscode` (and any other files/folders that are specific to your setup) to `.git/info/exclude`, and this folder will not be committed from.

For example, my `.git/info/exclude` looks like:
```
.env*
# IDE specific folder
.vscode
```
I believe it's also possible to set up a global git exclude file.

Reply:
:+1: